main.py

```diff
@@ -35,818 +35,792 @@
 import nemesis.arena_battles
 import nemesis.arenaPrep
 import nemesis.changes
+from game import get_gamestate
+
+def configuration_setup():
```

Comment:
Do you mind adding a comment here about what all you did? There is a lot here and it looks like it was mostly pulling things into functions and wrapping everything in `if __name__ == '__main__'`, but I'm not certain.

Reply:
Beyond what's in the issue itself or? Granted, the list was not very thorough.

Reply:
In short, I removed everything executing in the global context and put that in functions. The same sequence of execution as before can be found in the main function at the bottom, calling said functions. This means that said functions can be further refactored or split into separate files (similar to load_game) without fear of things breaking further.